(dp1
S'output'
p2
S'<class \'apiclient.errors.HttpError\'> <HttpError 404 when requesting https://www.googleapis.com/calendar/v3/calendars/terapiasholisticasdelser@gmail.com/events/rgbhqsqidldudedv9mpkv50th0@google.com returned "Not Found">'
p3
sS'layer'
p4
S'/home/wasuaje/Documentos/desarrollo/web2py6/applications/pycitas/controllers/default.py'
p5
sS'code'
p6
S'# -*- coding: utf-8 -*-\n### required - do no delete\n#\n# \n# sudo easy_install --upgrade google-api-python-client\n# sudo  easy_install rfc3339\n\n\n\ndef user(): return dict(form=auth())\ndef download(): return response.download(request,db)\ndef call():\n    session.forget()\n    return service()\n### end requires\ndef index():\n    form = SQLFORM.factory(\n        Field(\'Nombre\', requires=IS_NOT_EMPTY(),),\n        Field(\'Email\',),\n        Field(\'Tlf\',),\n        Field(\'Inicio\',\'datetime\'),\n        Field(\'Fin\',\'datetime\'),\n        \n        )\n    if form.process().accepted:\n        response.flash = \'Cita agregada %s \' % form.vars.Nombre        \n        session.your_name = form.vars.your_name\n        session.your_image = form.vars.your_image\n        insert_ajx(form.vars.Nombre,form.vars.Email,form.vars.Tlf,form.vars.Inicio,form.vars.Fin)\n    elif form.errors:\n        response.flash = \'form has errors\'\n\n    return dict(form=form)\n\n\ndef insert_ajx(nombre,email,tlf,inicio,fin):\n    rest="POST https://www.googleapis.com/calendar/v3/calendars/%s/events" % settings.google_calid\n    from rfc3339 import rfc3339\n    inicio=rfc3339(inicio)\n    fin=rfc3339(fin)\n    event = {\n        \'summary\': nombre,\n        \'location\': \'Caracas\',\n        \'start\': {\n            \'dateTime\': inicio,\n            \'timeZone\':"America/Caracas",\n            },\n        \'end\': {\n            \'dateTime\': fin,\n            \'timeZone\':"America/Caracas"\n            },\n        \'attendees\': [\n            {\n            \'email\': email,\n            # Other attendee\'s data...\n             },        \n        ],\n    }\n    from oauth2client import gce\n    import os\n    import httplib2\n    from apiclient.discovery import build    \n    from oauth2client.client import flow_from_clientsecrets\n    from oauth2client.file import Storage\n    from oauth2client.tools import run\n\n    jsonfile=os.path.join(request.folder,\'static\',\'client_secrets.json\')\n    storage = Storage(\'credentials.dat\')\n    credentials = storage.get()\n    \n    redirecturl=\'urn:ietf:wg:oauth:2.0:oob\'\n    flow = flow_from_clientsecrets(jsonfile,\n                               scope=\'https://www.googleapis.com/auth/calendar\',\n                               redirect_uri=redirecturl)\n\n    if credentials is None or credentials.invalid:\n        credentials = run(flow, storage)\n    http = httplib2.Http()\n    http = credentials.authorize(http)\n    \n    service = build(\'calendar\', \'v3\',http=http)\n\n    created_event = service.events().insert(calendarId=settings.google_calid, body=event).execute()\n    response.flash = created_event[\'id\']\n\ndef delete_ajx():\n    id=request.vars.id\n    from oauth2client import gce\n    import os\n    import httplib2\n    from apiclient.discovery import build    \n    from oauth2client.client import flow_from_clientsecrets\n    from oauth2client.file import Storage\n    from oauth2client.tools import run\n\n    jsonfile=os.path.join(request.folder,\'static\',\'client_secrets.json\')\n    storage = Storage(\'credentials.dat\')\n    credentials = storage.get()\n    \n    redirecturl=\'urn:ietf:wg:oauth:2.0:oob\'\n    flow = flow_from_clientsecrets(jsonfile,\n                               scope=\'https://www.googleapis.com/auth/calendar\',\n                               redirect_uri=redirecturl)\n\n    if credentials is None or credentials.invalid:\n        credentials = run(flow, storage)\n    http = httplib2.Http()\n    http = credentials.authorize(http)\n    \n    service = build(\'calendar\', \'v3\',http=http)\n    \n    created_event = service.events().delete(calendarId=settings.google_calid, eventId=id).execute()\n    response.flash = "Evento Eliminado %s" % created_event[\'id\']\n\n\ndef error():\n    return dict()\n\n@auth.requires_login()\ndef mymap():\n    rows=db(db.t_appointment.created_by==auth.user.id)(db.t_appointment.f_start_time>=request.now).select()\n    return dict(rows=rows)\n\n@auth.requires_login()\ndef mycal():\n    rows=db(db.t_appointment.created_by==auth.user.id).select()\n    return dict(rows=rows)\n\n@auth.requires_login()\ndef appointment_create():\n    form=crud.create(db.t_appointment,\n                     onvalidation=geocode2,\n                     next=\'appointment_read/[id]\')\n    return dict(form=form)\n\n@auth.requires_login()\ndef appointment_read():\n    record = db.t_appointment(request.args(0)) or redirect(URL(\'error\'))\n    form=crud.read(db.t_appointment,record)\n    return dict(form=form)\n\n@auth.requires_login()\ndef appointment_update():\n    record = db.t_appointment(request.args(0),active=True) or redirect(URL(\'error\'))\n    form=crud.update(db.t_appointment,record,next=\'appointment_read/[id]\',\n                     onvalidation=geocode2,\n                     ondelete=lambda form: redirect(URL(\'appointment_select\')),\n                     onaccept=crud.archive)\n    return dict(form=form)\n\n@auth.requires_login()\ndef appointment_select():\n    f,v=request.args(0),request.args(1)\n    query=f and db.t_appointment[f]==v or db.t_appointment\n    rows=db(query)(db.t_appointment.active==True).select()\n    return dict(rows=rows)\n\n@auth.requires_login()\ndef appointment_search():\n    form, rows=crud.search(db.t_appointment,query=db.t_appointment.active==True)\n    return dict(form=form, rows=rows)\n\nresponse._vars=response._caller(delete_ajx)\n'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/wasuaje/Documentos/desarrollo/web2py6/gluon/restricted.py", line 220, in restricted\n    exec ccode in environment\n  File "/home/wasuaje/Documentos/desarrollo/web2py6/applications/pycitas/controllers/default.py", line 163, in <module>\n  File "/home/wasuaje/Documentos/desarrollo/web2py6/gluon/globals.py", line 385, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/wasuaje/Documentos/desarrollo/web2py6/applications/pycitas/controllers/default.py", line 112, in delete_ajx\n    created_event = service.events().delete(calendarId=settings.google_calid, eventId=id).execute()\n  File "/usr/local/lib/python2.7/dist-packages/google_api_python_client-1.2-py2.7.egg/oauth2client/util.py", line 132, in positional_wrapper\n    return wrapped(*args, **kwargs)\n  File "/usr/local/lib/python2.7/dist-packages/google_api_python_client-1.2-py2.7.egg/apiclient/http.py", line 723, in execute\n    raise HttpError(resp, content, uri=self.uri)\nHttpError: <HttpError 404 when requesting https://www.googleapis.com/calendar/v3/calendars/terapiasholisticasdelser@gmail.com/events/rgbhqsqidldudedv9mpkv50th0@google.com returned "Not Found">\n'
p11
s.